restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/sql/call-detail-records.xml

```diff
@@ -201,4 +206,11 @@
     		end_conference_on_exit=#{end_conference_on_exit}, on_hold=#{on_hold}, ms_id=#{ms_id} 
     WHERE sid=#{sid};
   </update>
+
+  <update id="updateInCompleteCallDetailRecordsToCompletedByInstanceId" parameterType="string">
+    UPDATE "restcomm_call_detail_records"
```

Comment:
@anhntnguyen 
did we tested this file?
I think double quotes are not supported in this mariadb sql formats in this file.
i mean double quotes around table and column names.